src/AmsToMqttBridge.ino

```diff
@@ -462,6 +465,154 @@ void readHanPort() {
 						String msg;
 						serializeJson(json, msg);
 						mqtt.publish(config.getMqttPublishTopic(), msg.c_str());
+					// 
+					// Start DOMOTICZ
+					//
+					} else if(config.getMqttPayloadFormat() == 2) {
+						StaticJsonDocument<512> json;
+						hanToJson(json, data, hw, temperature);
```

Comment:
Drop converting data to JSON

Reply:
Agree.

Reply:
That part is the same as option 0. Left for now.  it has been nice to have the standard json as well when debugging.

Reply:
OK, I will remove it before release to preserve memory